cypress/e2e/awx/infrastructure/instance-groups/instanceGroups.cy.ts

```diff
@@ -153,7 +153,7 @@ describe(`Instance Groups`, () => {
       cy.filterTableByMultiSelect('name', arrayOfElementText);
       cy.get('tbody tr').should('have.length', 5);
       cy.getByDataCy('select-all').click();
-      cy.clickToolbarKebabAction('delete-selected-instance-groups');
+      cy.clickToolbarKebabAction('delete-instance-groups');
```

Comment:
The text is still `Delete selected instance groups` here causing a failure. I think the PR may need to be rebased.
https://github.com/ansible/ansible-ui/blob/a55433fa852e1df6b2a4a3050329af4a8bd671e1/frontend/awx/administration/instance-groups/hooks/useInstanceGroupActions.tsx#L71